Commerce 8347

riccardo-alberti opened the pull request.

jonmak08 commented:
/cc @SamZiemer

brianikim commented:
Hello @jonmak08 ,

I apologize for the delay in testing your change. Your source format works fine in fixing the issue.
Feel free to forward the pull and/or close this PR once you are ready to forward it to @ealonso for further review.

Lastly, thank you for updating my change. Thanks.

Sincerely,
Brian Kim

jonmak08 commented:
New Pull Request: https://github.com/ealonso/liferay-portal/pull/1591

jorgediaz-lr committed:
LPS-146617 Inline current _addFieldValueRequiredTerm method

jorgediaz-lr committed:
LPS-146617 Extract code to a new _addFieldValueRequiredTerm method

jorgediaz-lr committed:
LPS-146617 Simplify with only one isLegacyDDMIndexFieldsEnabled if block

jorgediaz-lr committed:
LPS-146617 Move localizable check to the createFieldValueQueryFilter method

Commit:
POSHI-216 Modify method to make it reusable

Commit:
POSHI-216 Update usage

Commit:
LPS-145031 Add the real TEXT and STRING data type sizes. When sql type STRING or TEXT were defined same type with VARCHAR in DB classes(SQLServerDB), DBInspector#hasColumnType can't use columnSize and columnDataType to check if it has ColumnType. For example, in the Layout table, the description field is defined as nvarchar(4000). When upgrade from 7.1.x to master, in com.liferay.portal.upgrade.v7_3_x.UpgradeLayout class, !hasColumnType(LayoutTable.TABLE_NAME, "description", "TEXT null") will return true because TEXT data type was defined as 'NVARCHAR' which same with description column 'NVARCHAR', and TEXT column size returns -1. We will miss the necessary upgrade.

Commit:
LPS-145031 Apply, get STRING and TEXT data type size to compare.

dantewang committed:
LPS-145031 SF, remove unnecessary unboxing

dantewang committed:
LPS-145031 SF, simplify, when the dataTypeSize is -1, just return it

dantewang committed:
LPS-145031 SF, simplify, avoid duplicate logic

Commit:
LPS-145480 Test dynamic collection combined assets shown

erickmonteiroo committed:
LPS-145342 (headless-admin-user-impl): Schema: Adds new endpoint getUserGroupPage

erickmonteiroo committed:
LPS-145342 (headless-admin-user): Implements getUserGroupsPage

miltonmc committed:
LPS-136111 Move Tokenizer to data-engine-js-components-web

Commit:
LPS-145031 Expose the issue. The test cases will fail under affected databases (DB2DB, HypersonicDB, OracleDB, PostgreSQLDB, SQLServerDB and SybaseDB).

Commit:
LPS-145031 packageinfo

dantewang committed:
LPS-145031 SF, also give -1 a meaningful name

izaera committed:
LPS-145792 Export react packages as ESM modules

With this, <script type="module"> nodes can make use of import declarations for
react packages, like this:

     import react from 'react';

Without any need to transpile or bundle anything, just plain ESM JavaScript
deployed to the portal.

Note that we declare ESM exports with a Java class. This is made on purpose,
because we don't want the import map to grow bigger than what is strictly
necessary, to avoid impacting performance.

If we provide an easier mechanism we run the risk of people abusing it and have
problems in the future. So, for now, I think an internal, unadvertised Java
class, is a good compromise.

izaera committed:
LPS-145792 gradle buildLang

Commit:
POSHI-243 Add method to get current UTC time

izaera committed:
LPS-145792 Rephrase labels

izaera committed:
LPS-145792 gradle buildLang

izaera committed:
LPS-145792 SF

Commit:
LPS-145381 Add Liferay Online tests to layout

Commit:
LPS-145381 Add Liferay Online tests to site-initializer-extender module

Commit:
LPS-145381 Fix Unmatched opening boundary in site-initializer-extender module

Commit:
LPS-145381 Fix Unmatched opening boundary in layout module

Commit:
LPS-145491 Handle case when there is no assignment children element in XML

aline-doleron committed:
LPS-145404 Check if transition default exists to set the defaultEdge

Commit:
LPS-145491 Implement function to parse assignment results from deserialization

sandrodw3 committed:
LPS-146271 Move container custom styles to general panel

sandrodw3 committed:
LPS-146271 Move row custom styles to general panel

sandrodw3 committed:
LPS-146271 Remove viewport label

sandrodw3 committed:
LPS-146271 Move contributed fragments config from styles to general

Commit:
LPS-145491 Add assignments to selectedItem

Commit:
LPS-145491 Display current assignments in node sidebar

javalosjr96 committed:
LPS-146237 Updated snapshot

cgoncas committed:
LPS-132509 When there is no subtype, the column subtype should be empty.

erickmonteiroo committed:
LPS-135342 (headless-admin-user-impl): Adds UserGroupEntityModel

erickmonteiroo committed:
LPS-145342 (headless-admin-user-impl): Implements getEntityModel

erickmonteiroo committed:
LPS-145342 (headless-admin-user): Baseline

Commit:
LPS-145491 Modify assignment options so it can be exported and used by other components

Commit:
LPS-145491 Move CurrentAssignments from Assignments.js file

Commit:
LPS-145491 Make section defined also by the currently selectedItem

Commit:
LPS-145491 Make SelectAssignment dropdown preselect option when coming from CurrentAssignments

Commit:
LPS-145268 remove default portal bundle url for InitBundleTask

Commit:
LPS-145268 Update project template tokens

garofalog committed:
COMMERCE-7600 fixed automatically map columns + sf

sandrodw3 committed:
LPS-146271 Adapt tests

javalosjr96 committed:
LPS-146237 Added createImageURL method to create source URL correctly

markocikos committed:
LPS-146459 Fire `sessionInitialized` custom event, when the session interval is initialized. This allows us to hook onto the component in race condition environment, when we cannot be certain that `Liferay.Session` is initialized.

mbowerman committed:
LPS-146481 setString and getString need to be used for the CLOB datatype
in PostgreSQL

erickmonteiroo committed:
LPS-145342 (headless-admin-user): Autogenerated: buildRest

erickmonteiroo committed:
LPS-145342 (headless-admin-user): Implements tests

iagochaves committed:
LPS-146474 Created RoundedGroupButton component

Commit:
LPS-145491 Create fetch methods to retrieve users and roles based on filter

iagochaves committed:
LPS-146474 Generic Table component refactor

Commit:
LPS-145491 Create utility function to return correct assignmentType

iagochaves committed:
LPS-146474 Added IconButton

iagochaves committed:
LPS-146474 Added DXP Activation Keys page

iagochaves committed:
LPS-146474 Added DXP Activation Keys styles

iagochaves committed:
LPS-146475 Populate table data with provisioning API

iagochaves committed:
LPS-146474 Fix package adding tooltip

Commit:
LPS-145491 Add function to retrieve extra data when diagram builder loads

Commit:
LPS-145491 Update selected item with role data and populate role component with current assignments

Commit:
LPS-145491 Update selected item with resource actions data and populate ResourceActions component with current assignments

iagochaves committed:
LPS-146474 Fix status bar table pagination

Commit:
LPS-145491 Update selected item with users data and populate component with current user assignments

Commit:
LPS-145491 Set sections with current user assignment data, if any

Commit:
LPS-145491 Display assignment details according to assignmentType

Commit:
LPS-145491 Add extra checks to show current assignments

markocikos committed:
LPS-146459 Add comment

markocikos committed:
LPS-146459 Fix toggling of `autoExtend` in runtime. Replace conditional attributes setting from Session instance creation to internal component logic. Trim out all unnecessary parts for session state tracking.

sandrodw3 committed:
LPS-146271 Add margin top to common styles when there are custom styles

sandrodw3 committed:
LPS-146271 Apply min height to Checkbox field

sandrodw3 committed:
LPS-146271 Remove cleanup

iagochaves committed:
LPS-146474 Fix imports & variables

victorg1991 committed:
LPS-146271 Apply min width only to common style checkbox

markocikos committed:
LPS-146459 Check first if the session should be expired. See LPS-137264

ruben-pulido committed:
LPS-146509 Display modified date in Fragment Collection

ruben-pulido committed:
LPS-146509 Display userId and userName in Fragment Collection

ruben-pulido committed:
LPS-146509 Show FragmentCollections and FragmentCollectionContributors in a compact way

garofalog committed:
COMMERCE-8315 language.properties

garofalog committed:
COMMERCE-8315  language translations

Commit:
COMMERCE-7600 batch-planner-web - remove redundant code - this is exactly the same

iagochaves committed:
LPS-146474 SF

achaparro committed:
LPS-145031 Avoid hardcoding max varchar size for every database since it can change or can be different for each database version

achaparro committed:
LPS-145031 Use a constant for size none

garofalog committed:
COMMERCE-8315  fix + sf

sandrodw3 committed:
LPS-146271 Remove unnecessary await from tests

Commit:
LPS-145491 Add lang file to process lang strings with variables.

bryceosterhaus committed:
LPS-135334 - Add proper return type for delegate

bryceosterhaus committed:
LPS-135334 - Migrate frontend-js-alert-support-web to typescript

Commit:
LPS-146478 Fix STATUS_CODE.success variable name

Commit:
LPS-146478 Add Download License Key

Commit:
LPS-146495 Fix styling for virtual cluster entry

matuzalemsteles committed:
LPS-144572 Update `@clayui/*` packages to v3.45.0

ruben-pulido committed:
LPS-146509 Display Search and Management Toolbar

ruben-pulido committed:
LPS-146509 Add groupId to FragmentCollectionItemSelectorViewDescriptor

ruben-pulido committed:
LPS-146509 Pass corresponding groupId to FragmentCollectionItemSelectorViewDescriptor from Global and Site Item Selector Views

ruben-pulido committed:
LPS-146509 Support ordering FragmentCollectionContributors by name

ruben-pulido committed:
LPS-146509 Support search for FragmentCollectionContributors

ruben-pulido committed:
LPS-146509 Support ordering FragmentCollections by name

ruben-pulido committed:
LPS-146509 Support search for FragmentCollections

bryceosterhaus committed:
LPS-135334 - Migrate frontend-js-collapse-support-web to typescript

bryceosterhaus committed:
LPS-135334 - Migrate frontend-js-tabs-support-web to typescript

bryceosterhaus committed:
LPS-135334 - Migrate frontend-js-dropdown-support-web to typescript

iagochaves committed:
LPS-146474 Fix table border styling

iagochaves committed:
LPS-146475 Adjust variables names

shuyangzhou committed:
LPS-146720 Merge BaseRawMetadataProcessor into TikaRawMetadataProcessor

shuyangzhou committed:
LPS-146720 Private as much as we can

shuyangzhou committed:
LPS-146720 Remove unused parameter

Commit:
LPS-145491 Add CSS class to truncate current assignments information in sidebar

ozysouza committed:
LPS-146271 Refactor container fragment macro and update usages

Commit:
LPS-146494 create type declaration for fetch from frontend-js-web

Commit:
LPS-146494 use fetch from frontend-js-web

ozysouza committed:
LPS-146271 Refactor grid fragment macro and update usages

Luizcarlosqueiroz committed:
LPS-146585 Add test for bug LPS-139951

shuyangzhou committed:
LPS-146720 Remove unused methods

shuyangzhou committed:
LPS-146720 Semantic versioning

shuyangzhou committed:
LPS-146720 Always bubble up exception to keep cause info.

shuyangzhou committed:
LPS-146720 SF

shuyangzhou committed:
LPS-146720 Only the InputStream version is used.

andrea-ale-sbarra committed:
COMMERCE-8236 commerce-product-definitions-web fixed bug on date cleaning

shuyangzhou committed:
LPS-146720 Use DefaultHandler for simpleness

shuyangzhou committed:
LPS-146720 Move Parser creation to java code.

Commit:
LPS-146446 Subscribe to message without checking permissions because is not needed

ozysouza committed:
LPS-146271 Update xpath to select Height on spacer fragment

ozysouza committed:
LPS-146271 Update usages for remaining fragments on basic components

marcelabc committed:
LPS-143347 Must not delete object field of relationship type

marcelabc committed:
LPS-143347 When deleting the relationship, the relationship field must be allowed to be deleted

marcelabc committed:
LPS-143347 buildService

rafaprax committed:
LPS-141305 Discard supplier approach

rafaprax committed:
LPS-141305 Prepare BundleSiteInitializer to receive the ServletContext and CommerceReferenceHolder references via DM

rafaprax committed:
LPS-141305 Remove the SiteInitializerRegistrar class and use the BundleSiteInitializer directly since it seems to be just a bridge between SiteInitializerExtention and the BundleSiteInitializer

rafaprax committed:
LPS-141305 Create an optional serviceDependency to the CommerceReferencesHolder. That way, when the CommerceReferencesHolder is available it will be injected directly into the BundleSiteInitializer instance.

erickmonteiroo committed:
LPS-146616 (account-admin-web): Conditionally check for add and update permissions

kevenleone committed:
LPS-146629 Add ClayUI Dependencies

kevenleone committed:
LPS-146629 Avoid Warning in Runtime

kevenleone committed:
LPS-146629 Create ListView Component

kevenleone committed:
LPS-146629 Create HeaderContext to manage title and tabs

kevenleone committed:
LPS-146629 Load data from GraphQL in some pages

kevenleone committed:
LPS-146629 Change Page Header when a project is selected

kevenleone committed:
LPS-146629 Populate Testray Object with data

Commit:
LPS-145128 Modification to UpperToobar's useState so that the title translation could be done and could be saved and published

bryceosterhaus committed:
LPS-135334 - Fix types in object-web

Commit:
LPS-145491 Rename assignment data retrieve and move it from DiagramBuilder

Commit:
LPS-145491 Rename serializer function to updateSelectedItem

kevenleone committed:
LPS-146629 Use Avatar Component inside of Sidebar

Co-authored-by: Wellington Barbosa <wellington.barbosa@liferay.com>

kevenleone committed:
LPS-146629 Remove TestFlow and border from Table

Co-authored-by: Wellington Barbosa <wellington.barbosa@liferay.com>

kevenleone committed:
LPS-146629 Create TestFlow Component and use mock data

Co-authored-by: Wellington Barbosa <wellington.barbosa@liferay.com>

kevenleone committed:
LPS-146629 Use Mock Data and Import Style of AVatar

Co-authored-by: Wellington Barbosa <wellington.barbosa@liferay.com>

xbrianlee committed:
LRQA-73038 Include MVCC backend tests in Publications test suite

Commit:
COMMERCE-7021 Fixes issue with image title

kevenleone committed:
LPS-146629 SF

shuyangzhou committed:
LPS-146720 Shortcut early

shuyangzhou committed:
LPS-146720 All fields are static, no need to invoke on metadata.

shuyangzhou committed:
LPS-146720 No need to create Metadata ahead of time.

shuyangzhou committed:
LPS-146720 Allow Metadata to be null

iagochaves committed:
LPS-146474 Renaming props variables RoundedGroupButtons

shuyangzhou committed:
LPS-146720 Shortcut early.

shuyangzhou committed:
LPS-146720 Avoid creating temp file when processing within portal jvm.

iagochaves committed:
LPS-146474 Generic table refinement

shuyangzhou committed:
LPS-146720 Prebake meta info's names and values to avoid runtime reflections.

shuyangzhou committed:
LPS-146720 The api consumer only cares the names of the meta info.

matuzalemsteles committed:
LPS-144572 Update `@clayui/core` package to v3.45.1

shuyangzhou committed:
LPS-146720 Finish the Property unwrapping at loading time.

iagochaves committed:
LPS-146474 Splitted constants folder in DXPActivationKeysTable container

petershin committed:
LPS-145268 Source formatting

ling-alan-huang committed:
LPS-105380 Fix logic to get portal version

marcelabc committed:
LPS-143347 baseline

brianchandotcom committed:
LPS-145792 Wordsmith

brianchandotcom committed:
LPS-145792 Rename

brianchandotcom committed:
LPS-145792 Rename

brianchandotcom committed:
LPS-145792 Wordsmith

brianchandotcom committed:
LPS-145792 Should it be ESM modules or ES modules?

brianchandotcom committed:
LPS-145792 regen

brianchandotcom committed:
e.g. vs. eg:

brianchandotcom committed:
LPS-145792 regen

cgoncas committed:
LPS-132509 New ContentDashboardItemBuilder, to make tests more legible.

brianchandotcom committed:
COMMERCE-8315 Wordsmith

brianchandotcom committed:
COMMERCE-8315 Regen

cgoncas committed:
LPS-132509 Add new test to check that when the subtype is Null, an empty subtype is returned.

DiegoHu97 committed:
LPS-146066: Text color accessibility issue on usages panel

brianchandotcom committed:
LPS-146629 upload

Commit:
LPS-145491 Rename usersData and rolesData to sectionsData

brianchandotcom committed:
LPS-145268 Wordsmith

brianchandotcom committed:
LPS-145268 prep next

lfbesada committed:
LPS-146673 Uses the proper object entry infoItemClassName and removes the variations.

ObjectEntry InfoFramework implementation has no variations, its variations are handled as different implementations

lfbesada committed:
LPS-146673 The templates should be displayed in the selector as a fieldset.

brianchandotcom committed:
LPS-145268 prep next

brianchandotcom committed:
Revert "LPS-145127 Removing the obsolete dl.file.entry.thumbnail.custom* properties"

This reverts commit e5c5c1c33164f1cfad2682bbdc15298f502aff6e.

andrea-ale-sbarra committed:
COMMERCE-8137 commerce-term-service added priority to search terms

andrea-ale-sbarra committed:
COMMERCE-8137 headless-commerce-admin-order-impl added priority field to sorts and filters

naraandrade committed:
LPS-146608 Add lang key

naraandrade committed:
LPS-146608 buildLang

naraandrade committed:
LPS-146608 Set component name explicitly

MarinhoFeliphe committed:
LPS-146605 Cover updateAsset with integration test

andrea-ale-sbarra committed:
COMMERCE-8137 commerce-term-web sort rows by priority

brianchandotcom committed:
LPS-146509 Simplify

naraandrade committed:
LPS-146608 Only components' names need to be started with uppercase letter

naraandrade committed:
LPS-146608 Create Notifications and NotificationsInfo components

MarinhoFeliphe committed:
LPS-146605 Set titleValue in asset entry title

naraandrade committed:
LPS-146608 Add notifications section to all node types

MarinhoFeliphe committed:
LPS-146605 Simplify

ozysouza committed:
LPS-146685 Make master page zip invalid

brianchandotcom committed:
LPS-146645 just hide it from the UI

Commit:
LPS-145491 Remove blank option and make assetCreator the default assignment

brianchandotcom committed:
LPS-143347 Wordsmith

epark18 committed:
LRQA-72928 Add test to assert a user with update permissions cannot view sites that another user belongs to

ling-alan-huang committed:
LPS-105380 Get rid of the prints that come from poshi-core

victorg1991 committed:
LPS-146697 Take into account blank page for the pagination

iagochaves committed:
LPS-146474 Fix variables

iagochaves committed:
LPS-146474 Fix getTooltipTitles

miltonmc committed:
LPS-136111 Move getDataDefinitionField to data-engine-js-components-web

Commit:
LPS-145128 Update test

Commit:
LRQA-73053 Quarantine CanApproveTranslation test

Commit:
LRQA-73053 SF

Commit:
LPS-145128 Title translation adjustment with default value

drewbrokke committed:
LPS-146707 portal-language-override-web: uses recalculated resultEnd

brianchandotcom committed:
LPS-146645 remove JSON

kenjiheigel committed:
POSHI-216 POSHI-243 Go back to original for now

kenjiheigel committed:
POSHI-243 Add timezone parameter to DateUtil

kenjiheigel committed:
POSHI-216 Add additional offset types

miltonmc committed:
LPS-136111 Move getDDMFormFieldSettingsContext to data-engine-js-components-web

Kvale1733 committed:
COMMERCE-8343 Fix location assert for product display page tests

miltonmc committed:
LPS-136111 Move getDataDefinitionFieldByFieldName to data-engine-js-components-web

carolmariaabb committed:
LPS-146717 Remove feature flag for one-to-many relationship between Native and Custom Objects

brianchandotcom committed:
LPS-105380 prep next

brianchandotcom committed:
LPS-105380 prep next

rafaellajordao committed:
LRQA-73013 revert path DROPDOWN_MENU_ITEM

Commit:
COMMERCE-7850 Add path OVERRIDE_DEFAULT_DISPLAY_PAGE_LAYOUT

Commit:
COMMERCE-7850 Add macro assertCategoryDisplayPageOverride

Commit:
COMMERCE-7850 Add macro chooseDisplayPageFromSidePanel

Commit:
COMMERCE-7850 Add macro overrideCategoryDisplayPage

Commit:
COMMERCE-7850 Add test to verify user can override a public default category display page with a private page

Commit:
COMMERCE-7852 Add test to verify user can override a private default category display page with another private page

brianchandotcom committed:
POSHI-216 POSHI-243 prep next

ruben-pulido committed:
LPS-146649 Show fragment label instead of lowercase name for contributed fragment collections

michaelhashimoto committed:
LRCI-2776 Rename the testray task to be called importTestrayResults

brianchandotcom committed:
LPS-146474 prep next

brianchandotcom committed:
LRQA-73038 #sfme

joshchong committed:
LRQA-73056 Add test for applying blueprint to content page via Low Level Search Options widget

brianchandotcom committed:
LRCI-2776 prep next

brianchandotcom committed:
LRCI-2776 prep next

brianikim committed:
COMMERCE-8347 Add methods to channel services

brianikim committed:
COMMERCE-8347 Use new finder in CPImageDataSetDataProvider

brianikim committed:
COMMERCE-8347 Autogenerated code (gw buildService)

riccardo-alberti committed:
COMMERCE-8347 Fix status

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

riccardo-alberti commented:
@brianikim there were just a couple of problems with your pr...
1) in the attachment data provider
2) in the service query where we check for status.
It should be good to go now
